tokio/src/runtime/mod.rs

```diff
@@ -597,4 +602,13 @@ cfg_rt! {
             }
         }
     }
+
+    cfg_metrics! {
+        impl Runtime {
+            /// TODO
```

Comment:
todo?

Reply:
I'm leaving it to a follow-up PR. Still unstable.